spec/lib/vmdb/config/validator_spec.rb

```diff
@@ -5,11 +5,6 @@
       expect(validator).to be_valid
     end
 
-    it "with a VMDB::Config" do
-      validator = described_class.new(VMDB::Config.new("vmdb"))
```

Comment:
If we are removing this, then we should also remove the support for it: https://github.com/ManageIQ/manageiq/blob/78f10b36dabbf67ed05d6553b9231767e165fa99/lib/vmdb/config/validator.rb#L10-L11

Otherwise, this test should be changed to do the OpenStruct trick like you did in the UI code.  However, I'd prefer just dropping it, and changing the UI to call the validator with the inner config.

Reply:
I removed support for `VMDB::Config` from both validator and activator.